test/mocha.opts

```diff
@@ -0,0 +1 @@
+./test/common
```

Comment:
Shouldn't this be `test/integration` to get `mocha` to pick up the `test/integration` directory?